cts/cts-scheduler.in

```diff
@@ -1053,7 +1054,6 @@ TESTS_64BIT = [
 class BuildVars(object):
     SBINDIR = "@sbindir@"
     BUILDDIR = "@abs_top_builddir@"
-    CRM_SCHEMA_DIRECTORY = "@CRM_SCHEMA_DIRECTORY@"
```

Comment:
I think it would be worthwhile to move all \@var\@ constants used by our python code into the new build options import. Of course they'll still need to be .in files if they're executable due to @\PYTHON@

Reply:
I think these are all taken care of now.